app/src/main/java/io/github/zadam/triliumsender/SendNoteActivity.kt

```diff
@@ -70,7 +130,13 @@ class SendNoteActivity : AppCompatActivity() {
             val client = OkHttpClient()
 
             val json = JSONObject()
-            json.put("title", noteTitle)
+
+            if (noteTitle.isEmpty()) {
+                // API does not allow empty note titles, so use a default value if the user doesn't set one.
+                json.put("title", "Note from Android")
```

Comment:
This confounded me while testing, had to login to my sync server to see that it 500ed on an empty note title.